### ISSUE TITLE
Commented HTML

### ISSUE THREAD

**Issue by dubcdr** [rchipka/node-osmosis#178]
I'm trying to scrape a site where the initial html file comes in with 2 tables i'm interested in. When I make the request and receive the initial html one of the tables is commented out

I know this from looking at the network tab. So imagine something basically equivalent to:

```
<div id="all_rushing_and_receiving">
<!---
  <table class="stats_table" id="rushing_and_receiving">...</table>
-->
</div>
```

but when I go into the element explorer before scrolling down or anything the table has been uncommented. 

I believe they have 2 tables for either mobile or widescreen and use vuejs to figure out which to uncomment. Is there anything I can do to make osmosis simulate this.

So basically I can access `#all_rushing_and_receiving table.stats_table in the browser immediately but osmosis doesn't fine it.

[Here](https://www.pro-football-reference.com/players/R/RodgAa00.htm#all_rushing_and_receiving) is the link of the page if you want to recreate

Thanks in advance for your help.

**Comment by improvenormal** [comment]
For anyone else who stumbles on this... this is actually more of an xpath selector concern than an osmosis concern. Using the example above, to get an html comment in xpath, you can do something like:
`#rushing_and_receiving/comment()`
